session_5/readme/README.md

```diff
@@ -0,0 +1,159 @@
+SCHEMA
```

Comment:
The Rails project has been added as a submodule, so delete the `.git` folder in `associations_validations`, add changes and push your code again.